src/appleseed/renderer/kernel/rendering/final/adaptivepixelrenderer.cpp

```diff
@@ -91,8 +91,8 @@ namespace
         {
             if (m_params.m_diagnostics)
             {
-                m_variation_aov_index = frame.create_extra_aov_image("variation");
-                m_samples_aov_index = frame.create_extra_aov_image("samples");
+                m_variation_aov_index = frame.create_extra_aov_image("variation", PixelFormatHalf);
```

Comment:
With these changes, the range and precision of the images used by the adaptive pixel renderer is much lower than before. This is going to affect the results of the adaptive sampling. I think we want to keep them as floats.

Reply:
I chanded the format because the tile_end maps values to colors.